projects/packages/image-cdn/src/class-image-cdn.php

```diff
@@ -988,6 +988,11 @@ public function filter_image_downsize( $image, $attachment_id, $size ) {
 	 * @return array An array of Photon image urls and widths.
 	 */
 	public function filter_srcset_array( $sources = array(), $size_array = array(), $image_src = array(), $image_meta = array(), $attachment_id = 0 ) {
```

Comment:
```suggestion
	public function filter_srcset_array( $sources = array(), $size_array = array(), $image_src = '', $image_meta = array(), $attachment_id = 0 ) {
```

I think this makes sense, but the `$image_src` should be a string. Unsure why we asked for an array looking at the code in `wp_calculate_image_sizes`

Reply:
✅